doc/netplan.md

```diff
@@ -624,6 +624,26 @@ wpasupplicant installed if you let the ``networkd`` renderer handle wifi.
           and ``adhoc`` (peer to peer networks without a central access point).
           ``ap`` is only supported with NetworkManager.
 
+     ``bssid`` (scalar)
+     :    If specified, directs the device to only associate with the given
+          access point.
+
+     ``band`` (scalar)
+     :    Possible bands are ``a`` (for 5GHz 802.11a) and ``bg``
+          (for 2.4GHz 802.11), do not restrict the 802.11 frequency band of the
+          network if unset (the default).
+
+     ``channel`` (scalar)
+     :    Wireless channel to use for the Wi-Fi connection. Because channel
+          numbers overlap between bands, this property takes effect only if
+          the ``band`` property is set, too.
+
+     ``seen-bssids`` (sequence of scalars)
+     :    A list of BSSIDs (each BSSID formatted as a MAC address) that have
+          been detected as part of the Wi-Fi network. This is a read-only
+          option, used only to track NetworkManager's state when using the
```

Comment:
I'm not sure what it means in this context for it to be "read-only".  It's clearly not read only from the perspective of the NM read-write plugin, else we would not need to store it.  If the meaning is that we will not emit it to NetworkManager via the netplan renderer, then I would say /that/ explicitly.

Reply:
Actually, the `wifi.seen-bssids` setting is deprecated. NetworkManager will not read this option or use it internally. Instead, it reads this data from it's internal state db at:
`/var/lib/NetworkManager/seen-bssids`
    
This is why we do not need to support this field for netplan.io. I added a commit to remove the corresponding code.